README.md

```diff
@@ -368,294 +396,260 @@ A device renderer instance can be configured using the `options` argument (objec
 }
 ```
 
-### `showPhoneBorder`
+### 🔌 Plugin Options
 
--   **Type:** `Boolean`
--   **Default:** `false`
--   **Details:**
-    Adds a mobile-style frame around the video to mimic the appearance of a smartphone screen.
-
-### `streamResolution`
-
-<img align="right" src="./doc/assets/ic_resolution.svg" alt="..."></img>
+#### <img style="vertical-align: middle" src="./doc/assets/ic_battery.svg"> `battery`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
--   **Compatibility:** `SaaS`
+-   **Toolbar name:** `Battery`
+-   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the video stream quality widget.
+    Enables or disables the battery widget. This widget can be used to set the battery level and state of the Android virtual device.
 
-### `streamBitrate`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_baseband.svg"> `baseband`
 
 -   **Type:** `Boolean`
 -   **Default:** `false`
--   **Compatibility:** `SaaS`
+-   **Toolbar name:** `Baseband`
+-   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the stream bitrate widget.
+    Enable or disable baseband (MMC/MNC) widget.
 
-### `touch`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_biometrics.svg"> `biometrics`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
--   **Compatibility:** `PaaS`, `SaaS`
+-   **Toolbar name:** `FingerPrint`
+-   **Compatibility:** `SaaS`, `PaaS`
 -   **Details:**
-    Enables or disables the touch events (fingers on screen). If you want to disable all VM interaction, please also disable `mouse` and `keyboard`.
+    Enable or disable fingerprints widget. This widget can be used to manage fingerprint reading requests. Available for Android 9 and above.
 
-### `mouse`
+#### <img style="vertical-align: middle" src="./doc/assets/ic_camera.svg"> `camera`
 
 -   **Type:** `Boolean`
 -   **Default:** `true`
+-   **Toolbar name:** `Camera`
 -   **Compatibility:** `PaaS`, `SaaS`
 -   **Details:**
-    Enables or disables the mouse events. If you want to disable all VM interaction, please also disable `touch` and `keyboard`.
+    Enables or disables the camera widget. This widget can be used to forward local webcam video to the Android virtual device. By default, if the `microphone` property is also true, then the default audio input will be used as well.
```

Comment:
```suggestion
    Enable or disable the camera widget. This widget can be used to forward the local webcam video stream to the Android virtual device. By default, if the `microphone` property is also true, then the default audio input will be used as well.
```

In some places, we have "Enable or disable", in some places "Enables or disables". Could be nicer to have consistency here, though it's not that critical 😅 .

Reply:
Looks like there's still one "enables or disables" left, for the navbar widget. (Also, it should probably be "navbar widget" instead of "navbar widgets", right? There's just one widget for the navbar?